ui-definitions/settings-video.php

```diff
@@ -35,13 +37,23 @@
 					'title'              => __( 'Video delivery', 'cloudinary' ),
 					'optimisation_title' => __( 'Video delivery', 'cloudinary' ),
 					'tooltip_text'       => __(
-						'Videos will be uploaded and delivered from Cloudinary.',
+						'If you turn this setting off, your videos will be delivered from WordPress.',
 						'cloudinary'
 					),
-					'description'        => __( 'Optimize and deliver videos on my site.', 'cloudinary' ),
+					'description'        => __( 'Sync and deliver videos from Cloudinary.', 'cloudinary' ),
 					'default'            => 'on',
 					'attributes'         => array(
-						'data-context' => 'image',
+						'data-context' => 'video',
+					),
+					'disabled'           => ! get_plugin_instance()->get_component( 'storage' )->is_local_full(),
```

Comment:
@pereirinha - If you place this in a callback, then in the component, you call it then, the settings API will be available.

Reply:
@DavidCramer 

I moved the approach to the suggested one.
Mind circling back?